delta/plugins/blazegraph/src/main/scala/ch/epfl/bluebrain/nexus/delta/plugins/blazegraph/model/BlazegraphView.scala

```diff
@@ -99,4 +105,17 @@ object BlazegraphView {
       override def index(view: IndexingViewResource): String =
         s"${config.prefix}_${uuid(view)}_${rev(view)}"
     }
+
+  @nowarn("cat=unused")
+  implicit private val blazegraphViewsEncoder: Encoder.AsObject[BlazegraphView] = {
+    implicit val config: Configuration                     = Configuration.default.withDiscriminator(keywords.tpe)
+    implicit val keyEncoder: KeyEncoder[TagLabel]          = KeyEncoder.instance(_.value)
```

Comment:
The tag part does not have to be encoded, since we don't care about it on the JsonLdEncoder.

Look at here: https://github.com/BlueBrain/nexus/blob/master/delta/plugins/elasticsearch/src/main/scala/ch/epfl/bluebrain/nexus/delta/plugins/elasticsearch/model/ElasticSearchView.scala#L126